mgmtsystem/openshift.py

```diff
@@ -100,3 +100,10 @@ def list_template(self):
             entity = Template(meta['name'], meta['namespace'])
             entities.append(entity)
         return entities
+
+    def list_image_openshift(self):
+        entities = []
+        entities_j = self.o_api.get('image')[1]['items']
+        for entity_j in entities_j:
+            entities.append(entity_j['metadata'])
```

Comment:
@jdemon519 But that doesn't answer my question. I didn't ask which images is it getting. I asked why does it not return the kubernetes.Image instance as the other methods do.